compiler/src/main/scala/org/scalaexercises/evaluator/Eval.scala

```diff
@@ -0,0 +1,195 @@
+/*
+ * scala-exercises-evaluator-compiler
+ * Copyright (C) 2015-2016 47 Degrees, LLC. <http://www.47deg.com>
+ */
+
+package org.scalaexercises.evaluator
+
+import java.io.{File, PrintWriter}
+import java.math.BigInteger
+import java.security.MessageDigest
+
+import org.scalaexercises.evaluator.Eval.CompilerException
+import org.xeustechnologies.jcl.{JarClassLoader, JclObjectFactory}
+
+import java.net.URLClassLoader
+import scala.reflect.internal.util.{AbstractFileClassLoader, Position}
+import scala.tools.nsc.Settings
+import scala.tools.nsc.io.{AbstractFile, VirtualDirectory}
+import scala.tools.nsc.reporters._
+
+/**
```

Comment:
We need attribution here to the fact that some of the code in this class comes from twitter-eval and include them in the copyright notice with links as well.